### PR TITLE
fix: build when some buildflags are disabled

### PR DESCRIPTION
**Pull request by deepak1556**
#### Description of Change

Fix build when the following flags are disabled

```
enable_basic_printing = false
enable_builtin_spellchecker = false
enable_desktop_capturer = false
enable_osr = false
enable_pdf = false
enable_pdf_viewer = false
enable_pepper_flash = false
enable_plugins = false
enable_print_preview = false
enable_widevine = false
enable_extensions = false
enable_electron_extensions = false
```

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: shell/browser/net/proxying_websocket.cc [hunk 1]
```diff
@@ -410,8 +405,7 @@ void ProxyingWebSocket::OnHeadersReceivedCompleteForAuth(
 
   auto continuation = base::BindRepeating(
```
Comment (deepak1556) [inline shell/browser/net/proxying_websocket.cc:406]: Auth requests for websocket proxy is not handled correctly, we don't expose any hooks through the webrequest module but still this is left in IO pending state. Will handle in a separate PR.

**Comment by alexeykuzmin** [comment]
@deepak1556 👍 
Are you going to backport it to `9-x-y`?

**Comment by deepak1556** [comment]
@alexeykuzmin yup will do

**Comment by deepak1556** [comment]
For history,

upstream issue when building with `enable_extensions=false` https://bugs.chromium.org/p/chromium/issues/detail?id=731689#c7

**Comment by deepak1556** [comment]
For `enable_plugins=false` https://chromium-review.googlesource.com/c/chromium/src/+/2173416/1

**Pull request merged**